### DISCUSSION TITLE
add Profiles like vscode

### DISCUSSION THREAD

**Discussion by krabhi1** [zed-industries/zed#14898] [Feature Requests]
is it possible to add [profiles](https://code.visualstudio.com/docs/editor/profiles) like vscode 
![image](https://github.com/user-attachments/assets/e81883de-795a-44f3-85e8-10d0c409c48d)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
